docs/guide/README.md

```diff
@@ -9,7 +9,7 @@ Welcome to MetaMask’s Developer Documentation. This documentation is for learn
 
 ## Why Metamask
 
-MetaMask was created out of the needs of creating more secure and usable Ethereum-based web sites. In particular, it handles account management and connecting the user to the blockchain.
+MetaMask was created to meet the needs of secure and usable Ethereum-based web sites. In particular, it handles account management and connecting the user to the blockchain.
```

Comment:
The change to this first line seems to change the point from 'MetaMask creates a more secure connection to sites' to 'MetaMask provides authors a secure way to interact with users'. Is that the intent here?

Reply:
I had difficulty interpreting the original sentence, to be honest. I remain unsure what "creates a more secure connection to sites" would mean.

Reply:
I think I extrapolated a little too hard there... The original doesn't mention connections at all. It implies MetaMask creates more secure _and usable_ *web sites*. In that light, your rewording aligns much more closely with what MetaMask actually does.